addons/mail/static/src/core/web/full_recipients_list.js

```diff
@@ -0,0 +1,60 @@
+import { _t } from "@web/core/l10n/translation";
+import { escape } from "@web/core/utils/strings";
+import { formatList } from "@web/core/l10n/utils";
+import { markup, Component } from "@odoo/owl";
+import { usePopover } from "@web/core/popover/popover_hook";
+
+import { RecipientList } from "@mail/core/web/recipient_list";
+import { SuggestedRecipientsList } from "@mail/core/web/suggested_recipient_list";
+import { Thread } from "@mail/core/common/thread_model";
+
+
+export class FullRecipientsList extends Component {
+    static template = "mail.FullRecipientsList";
+    static components = {
+        SuggestedRecipientsList,
+    };
+    static props = {
+        thread: { type: Thread },
+    };
+
+    setup() {
+        this.recipientsPopover = usePopover(RecipientList);
+    }
+
+    /**
+     * @returns {Markup}
+     */
+    getRecipientListToHTML() {
```

Comment:
Nitpicking: the less scrolling the better, because we see most template immediately at opening of file:

```js
static components = { SuggestedRecipientsList }; // in a single line
static props = { thread: { type: Thread } }; // again, in a single line
/** @returns {Markup} */ // also single line
```